buildSrc/src/test/java/com/osacky/flank/gradle/YamlWriterTest.kt

```diff
@@ -355,64 +352,64 @@ class YamlWriterTest {
   @Test
   fun writeResultsHistoryName() {
     val extension = emptyExtension {
-      resultsHistoryName = "androidtest"
+      resultsHistoryName.set("androidtest")
     }
 
-    assertEquals("  use-orchestrator: false\n" +
+    assertEquals("  results-history-name: androidtest\n" +
+        "  use-orchestrator: false\n" +
         "  auto-google-login: false\n" +
         "  record-video: true\n" +
         "  performance-metrics: true\n" +
         "  timeout: 15m\n" +
-        "  results-history-name: androidtest\n" +
         "  num-flaky-test-attempts: 0\n",
         yamlWriter.writeAdditionalProperties(extension))
   }
 
   @Test
   fun writeResultsBucket() {
     val extension = emptyExtension {
-      resultsBucket = "fake-project.appspot.com"
+      resultsBucket.set("fake-project.appspot.com")
     }
 
-    assertEquals("  use-orchestrator: false\n" +
+    assertEquals("  results-bucket: fake-project.appspot.com\n" +
+        "  use-orchestrator: false\n" +
         "  auto-google-login: false\n" +
         "  record-video: true\n" +
         "  performance-metrics: true\n" +
         "  timeout: 15m\n" +
-        "  results-bucket: fake-project.appspot.com\n" +
         "  num-flaky-test-attempts: 0\n",
         yamlWriter.writeAdditionalProperties(extension))
   }
 
   @Test
   fun writeResultsDir() {
     val extension = emptyExtension {
-      resultsDir = "resultsGoHere"
+      resultsDir.set("resultsGoHere")
     }
 
-    assertEquals("  use-orchestrator: false\n" +
+    assertEquals("  results-dir: resultsGoHere\n" +
+        "  use-orchestrator: false\n" +
         "  auto-google-login: false\n" +
         "  record-video: true\n" +
         "  performance-metrics: true\n" +
         "  timeout: 15m\n" +
-        "  num-flaky-test-attempts: 0\n" +
-        "  results-dir: resultsGoHere\n",
+        "  num-flaky-test-attempts: 0\n",
         yamlWriter.writeAdditionalProperties(extension))
   }
 
   @Test
   fun writeTestTargetsAndResultsHistoryName() {
     val extension = emptyExtension {
-      resultsHistoryName = "androidtest"
-      testTargets = listOf("class com.example.Foo")
+      resultsHistoryName.set("androidtest")
+      testTargets.set(listOf("class com.example.Foo"))
     }
 
-    assertEquals("  use-orchestrator: false\n" +
+    assertEquals("  results-history-name: androidtest\n" +
```

Comment:
can we keep the ordering the same here?

Reply:
Ok, I can rearrange that. This change is result of my refactoring in `YamlWriter` class where my intention was to keep related code in one place, example -- all logics related to default values is grouped together like:
```
// without default values
    val testShards = config.testShards
    val shardTime = config.shardTime
    val repeatTests = config.repeatTests
    val smartFlankGcsPath = config.smartFlankGcsPath
    val projectId = config.projectId
    val runTimeout = config.runTimeout
    val localResultsDir = config.localResultsDir

    if (testShards.isPresent) appendln("  max-test-shards: ${testShards.get()}")
    if (shardTime.isPresent) appendln("  shard-time: ${shardTime.get()}")
    if (repeatTests.isPresent) appendln(repeatTestsLine(repeatTests.get()))
    if (smartFlankGcsPath.isPresent) appendln("  smart-flank-gcs-path: ${smartFlankGcsPath.get()}")
    if (projectId.isPresent) appendln("  project: ${projectId.get()}")
    if (runTimeout.isPresent) appendln("  run-timeout: ${runTimeout.get()}")
    if (localResultsDir.isPresent) appendln("  local-result-dir: ${localResultsDir.get()}")
```

I think the best would be to refactor tests to check for presents for specific lines instead of whole, hardcoded string. What do you think?